homepage/models.py

```diff
@@ -53,6 +54,34 @@ class AppUser(models.Model):
     def __str__(self):
         return self.user.username
 
+    def toggle_user_activation(self):
+        self.user.is_active = not(self.user.is_active)
+
+    @staticmethod
+    # Creates a new appUser (without superuser permmisions) and saves it in the DB
+    def create_appUser(username, email, password):
+        appUser = AppUser()
+        user = User.objects.create_user(username, email, password)
+        appUser.user = user
+
+        appUser.save()
+        return appUser
+
+    @staticmethod
+    def get_all_appUsers():
+        appUsers = AppUser.objects.all()
+        return list(appUsers)
+
+    @staticmethod
+    def find_appUser(username_ToFind):
+        allAppUsers = AppUser.get_all_appUsers()

```

Comment:
Can I suggest a different approach? ```AppUser.objects.filter(username=username_ToFind)```
Pros:
  - You don't have to retrieve all users.
  - It is faster than iterating over all users.

Reply:
I had some issues using objects.get and objects.filter
if you can make it work in this method - let me know (on the new PR i'm opening)

For now I'm keeping it the old way